app/views/shared/_social_media_accounts.html.erb

```diff
@@ -1,15 +1,15 @@
 <% followus ||= false %>
 <% if socialable.social_media_accounts.any? %>
-  <section>
+  <section lang="en">
     <% unless followus %>
-      <h2 class="label"><%= t('social_media.follow_us') %></h2>
+      <h2 class="label">Follow on social media</h2>
     <% end %>
     <ul class="social-media-accounts js-hide-extra-social-media">
       <% socialable.social_media_accounts.each do |account| %>
         <%= content_tag_for(:li, account) do %>
           <%= link_to account.url,
                       class: "social-media-link #{account.service_name.parameterize}" do %>
-            <span class="connect">Connect with <%= account.socialable.display_name %> on</span> <%= account.display_name %>
+            <span class="connect"></span><%= account.display_name %>
```

Comment:
Is there a reason why we're removing the "Connect with" text?

Reply:
This is screen reader text that could be in two languages, so shouldn't be tagged as being in English. For example, on the [British Embassy in Seoul page](https://www.gov.uk/world/organisations/british-embassy-seoul.ko) it has "Connect with 주한영국대사관 on" because the `account.socialable.display_name` is somehow translated before it arrives at the template, but the rest of the sentence is not. This was a more consistent fix than trying to tag the display name in one language and the rest of the sentence in another.

Reply:
That makes sense 👍